Alicerce/Sources/Logging/Log+StringProvider.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  Log+StringProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+public extension Log {
+
+    public class StringProvider: LogProvider {
+
+        public var minLevel: Log.Level = .error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
+        public var output: String = ""
+        public var linefeed: String = "\n"
```

Comment:
You don't need to specify the types here. They can all be inferred (LogItemFormatter and Strings)